src/components/FocusZone/FocusZone.tsx

```diff
@@ -575,4 +581,31 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
 
   }
 
+  private _isElementInput(element: HTMLElement): boolean {
+    if (element.tagName && element.tagName.toLowerCase() === 'input') {
```

Comment:
This is checked just above. I think I'll clean up this section because this._activeElement are being used interchangeably and it can be confusing.